var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -250,6 +250,6 @@ def write_rpath_specs(self):
             for line in lines:
                 out.write(line + '\n')
                 if line.startswith('*link:'):
-                    out.write(r'-rpath {0}:{1} \n'.format(
```

Comment:
The spec line that resulted from that that version (e.g. fa4048aab6bb3ad3214daac069ea47fbe89109b9) looks like this:

```
*link:
-rpath /home/users/hartzelg/tmp/spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/gcc-5.4.0-fyuuybt7ln3dgr5627p44lhnrztpwoqu/lib:/home/users/hartzelg/tmp/spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/gcc-5.4.0-fyuuybt7ln3dgr5627p44lhnrztpwoqu/lib64 \
%{!static:--eh-frame-hdr} %{!mandroid|tno-android-ld:%{m16|m32|mx32:;:-m elf_x86_64}                    %{m16|m32:-m elf_i386}                    %{mx32:-m elf32_x86_64}   %{shared:-shared}   %{!shared:     %{!static:       %{rdynamic:-export-dynamic}       %{m16|m32:-dynamic-linker %{muclibc:/lib/ld-uClibc.so.0;:%{mbionic:/system/bin/linker;:/lib/ld-linux.so.2}}}       %{m16|m32|mx32:;:-dynamic-linker %{muclibc:/lib/ld64-uClibc.so.0;:%{mbionic:/system/bin/linker64;:/lib64/ld-linux-x86-64.so.2}}}       %{mx32:-dynamic-linker %{muclibc:/lib/ldx32-uClibc.so.0;:%{mbionic:/system/bin/linkerx32;:/libx32/ld-linux-x32.so.2}}}}     %{static:-static}};:%{m16|m32|mx32:;:-m elf_x86_64}                    %{m16|m32:-m elf_i386}                    %{mx32:-m elf32_x86_64}   %{shared:-shared}   %{!shared:     %{!static:       %{rdynamic:-export-dynamic}       %{m16|m32:-dynamic-linker %{muclibc:/lib/ld-uClibc.so.0;:%{mbionic:/system/bin/linker;:/lib/ld-linux.so.2}}}       %{m16|m32|mx32:;:-dynamic-linker %{muclibc:/lib/ld64-uClibc.so.0;:%{mbionic:/system/bin/linker64;:/lib64/ld-linux-x86-64.so.2}}}       %{mx32:-dynamic-linker %{muclibc:/lib/ldx32-uClibc.so.0;:%{mbionic:/system/bin/linkerx32;:/libx32/ld-linux-x32.so.2}}}}     %{static:-static}} %{shared: -Bsymbolic}}
```

That extra `\` is escaping the newline.  Not sure what that was considered useful, perhaps to make our addition stand out?